<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>371</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>589</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>589</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ginkgo USB-EasyScale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Init Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>78</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>78</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Device Index:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="deviceIndexComboBox">
                 <property name="minimumSize">
                  <size>
                   <width>86</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>6</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>7</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Channels(Hex):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="channelsLineEdit">
                 <property name="text">
                  <string>FFFF</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Init Data(Hex):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="initDataLineEdit">
                 <property name="text">
                  <string>FFFF</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>78</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>78</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>     ESDelay:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ESDelayLineEdit">
                 <property name="text">
                  <string>100</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>     ESDetect:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ESDetectLineEdit">
                 <property name="text">
                  <string>200</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>         ESWin:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ESWinLineEdit">
                 <property name="text">
                  <string>1000</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="minimumSize">
                  <size>
                   <width>78</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>78</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>      TStart:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TStartLineEdit">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>         TEos:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TEosLineEdit">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>       TValACK:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TValACKLineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="minimumSize">
                  <size>
                   <width>78</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>78</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>       TACKN:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TACKNLineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string> THighBitHigh:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="THighBitHighLineEdit">
                 <property name="text">
                  <string>20</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>   THighBitLow:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="THighBitLowLineEdit">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="minimumSize">
                  <size>
                   <width>78</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>78</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string> TLowBitHigh:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TLowBitHighLineEdit">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>   TLowBitLow:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TLowBitLowLineEdit">
                 <property name="text">
                  <string>20</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>     TimeError:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="TimeErrorLineEdit">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="minimumSize">
                <size>
                 <width>78</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>78</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>   TShutDown:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="TshutDownLineEdit">
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>5000</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="initPushButton">
               <property name="text">
                <string>Init</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Send Data </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_27">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_28">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Channels(Hex):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dataChannelsLineEdit">
               <property name="text">
                <string>FFFF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="minimumSize">
                <size>
                 <width>78</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>78</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>DevAddr(Hex):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="devAddrLineEdit">
               <property name="text">
                <string>72</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_30">
                 <property name="text">
                  <string>RFA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="RFALineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>      SubAddr:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="subAddrLineEdit">
               <property name="maximumSize">
                <size>
                 <width>95</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_32">
               <property name="minimumSize">
                <size>
                 <width>78</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>78</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>     Data:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dataLineEdit">
               <property name="minimumSize">
                <size>
                 <width>114</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>114</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enterEsModeCheckBox">
               <property name="text">
                <string>EnterESMode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="writeDataPushButton">
               <property name="text">
                <string>WriteData</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
